bots/learn/extractor.py

```diff
@@ -43,20 +43,30 @@
 # some cluster seeds
 TRACKER_SPARSE = 100
 
-# TODO: We should be able to detect these automatically and ignore them
-# But for now this is a pragmatic hack to reduce noise and processing time
-NOISE = {
-    "Wrote file": re.compile("Wrote.*\.(png|html|log)"),
-    "Journal extracted": re.compile("Journal extracted to.*\.log"),
-    "Core dumps downloaded": re.compile("Core dumps downloaded to.*\.core"),
-    "not ok": re.compile("^not ok.*"),
-    "ok": re.compile("^ok.*"),
-    "# Flake": re.compile("# Flake.*"),
-    'File "\\1"': re.compile('File "/[^"]+/([^/]+)"'),
-    "### ": re.compile('#{3,80}\s+'),
-}
-
-DIGITS = re.compile('\d+')
+NUMBERS = (
+    # 512 bit hashes
+    ("xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx", re.compile('[0-9a-f]{128}')),
+    ("XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX", re.compile('[0-9A-F]{128}')),
```

Comment:
<nitpick> IMHO `string * repeat` is more intuitive than `repeat * string`.

Reply:
Need to use the numbers directly ... rather than dealing with bits, due the regexes ... but i've fixed this up in https://github.com/cockpit-project/cockpituous/pull/208

Reply:
Does that PR obsolete this one?